### PR TITLE
Update main.cpp

### PR DESCRIPTION
**Pull request by akshayMore2018**
few of the code refactoring changes

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1,24 +1,27 @@
 #include "SFML/Graphics.hpp"
 #include <iostream>
 
-
+//
```
Comment (akshayMore2018) [inline main.cpp:4]: looks like a constant value to me, add const key word, what if someone accidentally changes it somewhere?

File: main.cpp [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
 int WIDTH = 600;
 int HEIGHT = 600;
 
 int main()
 {
+	//
```
Comment (akshayMore2018) [inline main.cpp:10]: Make a variable Name/Title and then pass that in. What if I need to use the "Snake" somewhere else, will I always create new string "Snake"?

File: main.cpp [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
 	sf::RenderWindow window(sf::VideoMode(WIDTH, HEIGHT), "Snake");
-
+	//
```
Comment (akshayMore2018) [inline main.cpp:12]: Make a variable that easy to manage , pass that in instead of hardcoded values.

File: main.cpp [hunk 4]
```diff
@@ -1,24 +1,27 @@
...
 	sf::RectangleShape rectangle(sf::Vector2f(0.0f,0.0f));
-
+	//
```
Comment (akshayMore2018) [inline main.cpp:14]: Same as above

File: main.cpp [hunk 5]
```diff
@@ -1,24 +1,27 @@
...
 	rectangle.setSize(sf::Vector2f(20.0f,20.0f));
-
+	//
 	rectangle.setFillColor(sf::Color(255,0,0));
-
+	//
 	rectangle.setPosition(0,0);
-
+	//
```
Comment (akshayMore2018) [inline main.cpp:20]: Are you sure speed is to be an integer , can speed not be float?

File: main.cpp [hunk 6]
```diff
@@ -1,24 +1,27 @@
...
 	int Speed = 10;
+	//
 	int Movex = 0;
```
Comment (akshayMore2018) [inline main.cpp:23]: either use camelcase or Pascal case when naming variables, this is so disturbing to look at.

**Review by akshayMore2018 — COMMENTED** [review]
Just a few comments that need  to be taken care of

File: main.cpp (hunk 1 above)
Comment (abhayMore) [reply main.cpp:4]: Done

File: main.cpp (hunk 2 above)
Comment (abhayMore) [reply main.cpp:10]: Done

File: main.cpp (hunk 3 above)
Comment (abhayMore) [reply main.cpp:12]: Done

File: main.cpp (hunk 4 above)
Comment (abhayMore) [reply main.cpp:14]: Done

File: main.cpp (hunk 5 above)
Comment (abhayMore) [reply main.cpp:20]: Done

File: main.cpp (hunk 6 above)
Comment (abhayMore) [reply main.cpp:23]: Done

**Pull request closed without merging**